tests/unit/nupic/algorithms/anomaly_test.py

```diff
@@ -59,38 +59,38 @@ def testComputeRawAnomalyScorePartialMatch(self):
 
 
   def testComputeAnomalyScoreNoActiveOrPredicted(self):
-    anomalyComputer = anomaly.Anomaly()
+    anomalyComputer = anomaly.Anomaly(claBurnInPeriod=0)
     score = anomalyComputer.compute(array([]), array([]))
     self.assertAlmostEqual(score, 0.0)
 
 
   def testComputeAnomalyScoreNoActive(self):
-    anomalyComputer = anomaly.Anomaly()
+    anomalyComputer = anomaly.Anomaly(claBurnInPeriod=0)
     score = anomalyComputer.compute(array([]), array([3, 5]))
     self.assertAlmostEqual(score, 0.0)
 
 
   def testComputeAnomalyScorePerfectMatch(self):
-    anomalyComputer = anomaly.Anomaly()
+    anomalyComputer = anomaly.Anomaly(claBurnInPeriod=0)
     score = anomalyComputer.compute(array([3, 5, 7]), array([3, 5, 7]))
     self.assertAlmostEqual(score, 0.0)
 
 
   def testComputeAnomalyScoreNoMatch(self):
-    anomalyComputer = anomaly.Anomaly()
+    anomalyComputer = anomaly.Anomaly(claBurnInPeriod=0)
     score = anomalyComputer.compute(array([2, 4, 6]), array([3, 5, 7]))
     self.assertAlmostEqual(score, 1.0)
 
 
   def testComputeAnomalyScorePartialMatch(self):
-    anomalyComputer = anomaly.Anomaly()
+    anomalyComputer = anomaly.Anomaly(claBurnInPeriod=0) # disable the burn-in period for the tests
     score = anomalyComputer.compute(array([2, 3, 6]), array([3, 5, 7]))
     self.assertAlmostEqual(score, 2.0 / 3.0)
 
 
   def testAnomalyCumulative(self):
     """Test cumulative anomaly scores."""
-    anomalyComputer = anomaly.Anomaly(slidingWindowSize=3)
+    anomalyComputer = anomaly.Anomaly(slidingWindowSize=3, claBurnInPeriod=0)
```

Comment:
or should `0` here mean: "completely ignore :fire: in" ? and always return `isReady()` True?